guides/source/developers/customizations/customizing-attributes.html.md

```diff
@@ -0,0 +1,92 @@
+# Customizing Model Attributes
+
+This guide covers how to add editable attributes to models in Solidus.
+We will cover an example of adding a new attribute to the `Spree:Taxon` model 
+to allow for an additional text area.
+
+-   Adding attribute columns to the database
+-   Adding attribute fields to the admin panel
+-   Allowing custom attributes to be saved
+
+## Adding Columns to the Database
+
+Columns can be added to Solidus database tables using standard Rails migrations
+[standard Rails migrations][standard_rails_migrations].
+
+If you wanted to add a `more_information` attribute, you could run the 
+following command:
+
+```bash
+rails generate migration AddMoreInformationToTaxons
+``` 
+
+This would create a migration file starting with a timestamp 
+`db/migrate/20190925192026_add_more_information_to_taxons.rb`. Then update the 
+file with the following contents to add a new text type column 
+`more_information` to the table:
+
+```ruby
+class AddMoreInformationToTaxons < ActiveRecord::Migration[5.2]
+    def change
+        add_column :spree_taxons, :more_information, :text
+    end
+end
```

Comment:
Can you please use 2 indentation spaces instead of 4? Just to be consistent with the rest of the guides and the code style we use in the codebase.

Reply:
Fixed this, didn't realize it was auto formatting my markdown differently automatically in my editor. Sorry about that.